Libs/MRML/Core/vtkMRMLDisplayableNode.cxx

```diff
@@ -292,6 +292,7 @@ void vtkMRMLDisplayableNode::SetDisplayVisibility(int visible)
       displayNode->SetVisibility(visible);
       }
     }
+  this->Modified();
```

Comment:
If only the visibility is changed then the displayable node should not be modified, because that may be a very expensive operation. For example, if volume rendering is enabled then a lot of computation is done when the volume is changed, while nothing is recomputed if only the visibility is changed. It's convenient but misleading to to have this SetDisplayVisibility method in the displayable node.